app/Nova/Attendance.php

```diff
@@ -101,7 +101,7 @@ public function fields(Request $request): array
                     Team::class,
                 ]),
 
-            BelongsTo::make('Recorded By', 'recorded', 'App\Nova\User')
+            BelongsTo::make('Recorded By', 'recorded', User::class)
```

Comment:
Since App\User and App\Nova\User are different, we need to make sure this is referencing the right one.

Reply:
each of those files is in the `\App\Nova` namespace so if you say `Payment` that means `\App\Nova\Payment`

Reply:
each of those files is in the `\App\Nova` namespace so if you say `Payment` that means `\App\Nova\Payment`